src/services/DatasetService.js

```diff
@@ -0,0 +1,96 @@
+import datasette from './datasette.js'
+import logger from '../utils/logger.js'
+
+export async function getLatestDatasetResourceForLpa (dataset, lpa) {
+  const sql = `
+    SELECT rle.resource
+    FROM reporting_latest_endpoints rle
+    LEFT JOIN resource_organisation ro ON rle.resource = ro.resource
+    LEFT JOIN organisation o ON REPLACE(ro.organisation, '-eng', '') = o.organisation
+    WHERE REPLACE(ro.organisation, '-eng', '') = '${lpa}'
+    AND rle.pipeline = '${dataset}'`
+
+  const resources = await datasette.runQuery(sql)
+
+  return resources?.formattedData[0]
+}
```

Comment:
I believe we might have a function for this already in performanceDbApi.getLatestResource(). if so should we remove one of them?

as another observation. our use of datasette.js and also DatasetService isn't ideal atm. as we will be moving to the performance db anyway. and something like this might be in there?

Reply:
Thanks @GeorgeGoodall-GovUk - I'll remove my code and use the one from the other service 👍